Gemfile

```diff
@@ -26,6 +26,7 @@ gem 'rest-client', '~> 2.0.2'
 gem 'rubyzip', '~> 1.2'
 gem 'sass-rails', '~> 5.0'
 gem 'sentry-raven'
+gem 'sidekiq-prometheus-exporter'
```

Comment:
Might be nice to specify a `~>` version here.

Reply:
Good point.  I've updated this with a version.